selfdrive/locationd/test/test_glonass_kaitai.cc

```diff
@@ -0,0 +1,368 @@
+#include <iostream>
+#include <vector>
+#include <bitset>
+#include <cassert>
+#include <cstdlib>
+#include <ctime>
+
+#include "catch2/catch.hpp"
+#include "../generated/glonass.h"
```

Comment:
paths should always be relative to the openpilot root
```suggestion
#include "selfdrive/locationd/generated/glonass.h"
```